juri/Mobile_Signal.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
You don't need this, we don't have any mobile icon in the design. Please, remove